src/hooks-worker-client.coffee

```diff
@@ -9,23 +9,19 @@ logger = require './logger'
 which = require './which'
 
 
-HOOK_TIMEOUT = 5000
-
-CONNECT_TIMEOUT = 1500
-CONNECT_RETRY = 500
-AFTER_CONNECT_WAIT = 100
-
-TERM_TIMEOUT = 5000
-TERM_RETRY = 500
-
-HANDLER_HOST = 'localhost'
-HANDLER_PORT = 61321
-HANDLER_MESSAGE_DELIMITER = "\n"
-
-
 class HooksWorkerClient
   constructor: (@runner) ->
-    @language = @runner.hooks?.configuration?.options?.language
+    options = @runner.hooks.configuration.options
+    @language = options.language
+    @timeout = options['hook-worker-timeout'] || 5000
+    @connect_timeout = options['hook-worker-connect-timeout'] || 1500
+    @connect_retry = options['hook-worker-connect-retry'] || 500
+    @after_connect_wait = options['hook-worker-after-connect-wait'] || 100
+    @term_timeout = options['hook-worker-term-timeout'] || 5000
+    @term_retry = options['hook-worker-term-retry'] || 500
+    @handler_host = options['hook-worker-handler-host'] || 'localhost'
+    @handler_port = options['hook-worker-handler-port'] || 61321
+    @handler_message_delimiter = options['hook-worker-handler-message-delimiter'] || '\n'
```

Comment:
Just remove the options overriding, please. The configuration can totally be in the constructor.